client/src/views/admin-academy/ClassSubject.vue

```diff
@@ -13,14 +13,30 @@
       <div>
         <div class="card-body">
           <h2 class="h4 mb-2 text-gray-800">About this classs</h2>
-          <p>Time: {{ listClasses.Time }}</p>
-          <p>Capacity: {{ listClasses.Capacity }}</p>
-          <p>
-            Status:
-            <b-badge :variant="listClasses.Status === 'Open' ? 'success' : 'danger'">{{
-              listClasses.Status
-            }}</b-badge>
-          </p>
+          <div class="row">
+            <div class="col">
+              <p>Time: {{ listClasses.Time }}</p>
+              <p>Start Date: {{ listClasses.StartDate }}</p>
+              <p>End Date: {{ listClasses.EndDate }}</p>
+            </div>
+            <div class="col">
+              <p>Capacity: {{ listClasses.Capacity }}</p>
+              <p>
+                Status:
+                <b-badge :variant="listClasses.Status === 'Open' ? 'success' : 'danger'">{{
+                  listClasses.Status
+                }}</b-badge>
+              </p>
+              <p>
+                Teacher:
+                <router-link :to="`/academy/teachers/${listClasses.TeacherUsername}/subjects`">
```

Comment:
![image](https://user-images.githubusercontent.com/53574829/75644528-292d8700-5c75-11ea-9ae6-75f50c76c712.png)

Ấn vào tên teacher bị not found 404 quang à